packages/app/src/systems/Settings/pages/ChangePassword/ChangePassword.tsx

```diff
@@ -54,8 +56,19 @@ export function ChangePassword() {
     setError,
     setValue,
     formState: { isValid },
+    trigger,
   } = form;
 
+  const debouncedValidate = debounce(() => {
+    trigger('confirmPassword');
+  }, 500);
```

Comment:
This looks a bit concerning as it will be render on every render, possibly creating memory leaks would be good to have this inside a useCallback without dependencies that would enable to clean the debounce on each render.

```suggestion
  const debouncedValidate = useCallback(debounce(() => {
    trigger('confirmPassword');
  }, 500), []);
```